src/pages/[platform]/build-a-backend/add-aws-services/logging/view-logs/index.mdx

```diff
@@ -0,0 +1,35 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'View logs',
+  description: 'Learn how to view logs in Amazon CloudWatch',
+  platforms: [
+    'swift',
+    'android'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+Log messages are sent to the Amazon CloudWatch log group that you created and specified in the [initial set up](/gen1/[platform]/build-a-backend/more-features/logging/set-up-logging/). Each user log messages can be found in user-specific log stream in the log group. Log stream names are generated based on the user's device identifier and a user identifier.
+For an authenticated user, the log stream name is `<device-identifier>.<user-identifier>`.  For an unauthenticated user, the log stream name is `<device-identifier>.guest`.
+
+The device identifier is the unique identifier provided by the user's hardware device and OS. You can use the Amplify Auth category to [retrieve the user identifier](/gen1/[platform]/build-a-backend/more-features/logging/configure-user/#retrieve-userids-with-amplify-auth) for a specific user if needed. You can also retrieve the user identifier by visiting the Amazon Cognito console and inspecting the `User ID` in User pools.
```

Comment:
Don't know too much about the content, but should these be linking to gen1? Just wanted to verify

Reply:
Yes...Updated